app/src/main/AndroidManifest.xml

```diff
@@ -1,20 +1,26 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="com.github.palFinderTeam.palfinder" >
+    package="com.github.palFinderTeam.palfinder">
 
     <application
         android:allowBackup="true"
         android:icon="@mipmap/ic_launcher"
         android:label="@string/app_name"
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
-        android:theme="@style/Theme.PalFinder" >
+        android:theme="@style/Theme.PalFinder">
```

Comment:
don't forget to add a parent to the MeetupView and Meetup creation activities, so that we can navigate easily between views